src/main/resources/META-INF/lsp.xml

```diff
@@ -9,11 +9,11 @@
   -->
 
 <idea-plugin>
-    <extensions defaultExtensionNs="open-liberty.intellij">
+    <extensions defaultExtensionNs="com.redhat.devtools.lsp4ij">
         <!-- MicroProfile LS -->
         <server id="lsp4mp"
                 name="MicroProfile"
-                factoryClass="io.openliberty.tools.intellij.lsp4mp.lsp.MicroProfileLanguageServerFactory"
+                factoryClass="io.openliberty.tools.intellij.lsp4mp.lsp.MircoProfileLanguageServerFactory"
```

Comment:
This bug was fixed in other branches, please leave it MicroProfileLanguageServerFactory.